go.mod

```diff
@@ -11,4 +11,7 @@ require (
 	github.com/stretchr/testify v1.5.1
 )
 
+replace (
+	github.com/edgexfoundry/go-mod-core-contracts => ../go-mod-core-contracts
+)
```

Comment:
This shouldn't have been in the commit?

Reply:
Was needed for testing. Left in as a reminder that core-contracts version will need to be bumped when that PR is merged.